### PR TITLE
Project-1: Initialisation

### PR DESCRIPTION
**Pull request by SamanAtashi**
# Project 1 : Initialisation
Features introduced are as follows:
1. React-Redux
2. ducks pattern
3. React Router

### CODE REVIEW

File: src/components/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <p>Under construction</p>;
+
+export default Categories;
```
Comment (dasileker) [inline src/components/Categories.jsx]: - [x] This component should be  done as `Route or Link` see the screenshot 👇.

kklkl

File: src/components/Books.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Book from './Book';
+
+const Books = () => (
+  <ul>
+    <Book />
+  </ul>
+);
```
Comment (dasileker) [inline src/components/Books.jsx]: - [x] also This should be the same as the category `Routes or Link`

File: src/components/Book.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    { id: 1, name: 'book1' },
+    { id: 2, name: 'book2' },
+  ];
+
+  return books.map((book) => (
+    <li key={book.id}>
+      {book.name}
+      <button type="button">remove</button>
+    </li>
+  ));
```
Comment (dasileker) [inline src/components/Book.jsx]: - [x] This function should have a propTypes and  validate it.

lmlml

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @SamanAtashi** 

Good job so far!
You are almost there, you need just a little and fix some issues that are required to change 🔴

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oxenprogrammer — APPROVED** [review]
Hi **@SamanAtashi** , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**